examples/Emulator/GaussianProcess/Manifest.toml

```diff
@@ -34,40 +103,58 @@ uuid = "83423d85-b0ee-5818-9007-b63ccbeb887a"
 version = "1.16.1+1"
 
 [[CalibrateEmulateSample]]
-deps = ["Conda", "Distributions", "DocStringExtensions", "EnsembleKalmanProcesses", "GaussianProcesses", "LinearAlgebra", "PyCall", "Random", "ScikitLearn", "Statistics", "StatsBase"]
-path = "/Users/tsj/Documents/climate/clima/CalibrateEmulateSample.jl"
+deps = ["AbstractMCMC", "AdvancedMH", "Conda", "Distributions", "DocStringExtensions", "EnsembleKalmanProcesses", "GaussianProcesses", "LinearAlgebra", "MCMCChains", "Printf", "PyCall", "Random", "ScikitLearn", "Statistics", "StatsBase"]
+path = "../../.."
```

Comment:
@odunbar this is what to look out for (and is the result of `dev`ing)

Reply:
Gotcha!